assets/js/blocks/product-gallery/inner-blocks/product-gallery-pager/index.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { isExperimentalBuild } from '@woocommerce/block-settings';
+
+/**
+ * Internal dependencies
+ */
+import { ProductGalleryBlockIcon } from './icons';
+import { Edit, Save } from './edit';
+import metadata from './block.json';
+import './style.scss';
+import './editor.scss';
+
+if ( isExperimentalBuild() ) {
+	// @ts-expect-error: `metadata` currently does not have a type definition in WordPress core
+	registerBlockType( metadata, {
+		icon: ProductGalleryBlockIcon,
+		edit: Edit,
+		save: Save,
```

Comment:
I believe `Save` here is unnecessary as the inner blocks get saved via the parent's Save function.

```suggestion
		save() {
			return null;
		},
```

Reply:
You're right! I just removed that function!